lnrpc/rpc.swagger.json

```diff
@@ -3646,6 +3646,10 @@
             "$ref": "#/definitions/lnrpcFeature"
           },
           "description": "/ Features advertised by the remote peer in their init message."
+        },
+        "latest_error": {
+          "$ref": "#/definitions/lnrpcTimestampError",
+          "description": "Latest error is the last error we received from the peer. This error is\ntracked across peer connections, but is not currently persisted across lnd\nrestarts."
```

Comment:
nit: Instead of "latest error is the last error..", just "The latest error with timestamp that we received from the peer".